packages/kbn-typed-react-router-config/src/create_router.test.tsx

```diff
@@ -241,5 +277,16 @@ describe('createRouter', () => {
         } as any);
       }).toThrowError();
     });
+
+    it('applies defaults', () => {
+      expect(() => {
+        router.link('/traces', {
+          query: {
+            rangeTo: 'now',
+            aggregationType: 'avg',
+          },
+        } as any);
+      }).not.toThrowError();
```

Comment:
Don't you need to make an expectation against the generated link to ensure that defaults are applied? Like:

```ts
        const link = router.link('/traces', {
          query: {
            rangeTo: 'now',
            aggregationType: 'avg',
          },
        } as any);
		expect(link).toEqual(...)
```

Reply:
good one, updated the tests 👍